| Dataset     | ENA study accession | PMID (or DOI)                | Cancer type                     | Sample count | Protocol                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
| ----------- | ------------------- | ---------------------------- | ------------------------------- | ------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| Havas et al | PRJEB13586          | 28504653                     | Breast Cancer                   | 14           | Neoadjuvant treated patients were selected from the Genomstudy of the University Women\`s Clinic Heidelberg (2009-2016). This study includes primary breast cancer cases that were newly diagnosed at the University women's clinic of Heidelberg and had given their informed consent for participating in this study. This study was approved by the Ethical Committee of the Medical Faculty in Heidelberg. All cases were female and Caucasian. Fresh core needle biopsies and fresh tumor tissue were examined by a pathologist, snap-frozen in liquid nitrogen and stored at -80C within 15 min after sampling. Total RNA as well as DNA and protein were extracted by applying the QIAGEN All Prep Kit. All eluates were stored at -80C until usage. A sample swap most probably occurred for patient 207, so 207-S is actually in the T condition and 207-T in the S condition.           |
| Xu et al    | PRJNA553624         | 32838810                     | Breast Cancer                   | 12           | Total RNA were extracted from peripheral blood and library were constructed for the illumina Xten sequencing. We did transcriptomic sequencing of adjacent normal tissues and cancer tissues from 6 breast cancer patients. Total RNA were extracted and library were constructed for the illumina Xten sequencing.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
| Zhang et al | PRJNA636799         | NA                           | Cervical Cancer                 | 68           | Investigate the transcriptomic differences between HPV-positive and HPV-negative primary cervical tumors. Overall design: Whole transcriptome sequencing was performed on 68 pre-treatment, primary cervical cancer samples from cervical cancer patients treated at the Washington University School of Medicine. A minimum of 0.5µg of RNA with RIN > 7 was extracted for library preparation. Sequencing was performed using Illumina HiSeq 3000, 1x50nt, ~40 million reads per sample), and aligned to GRCh38 using STAR 2.7.0f.                                                                                                                                                                                                                                                                                                                                                              |
| Yao et al   | PRJNA396588         | 10.1158/2159-8290.CD-17-0375 | Clear Cell Renal Cell Carcinoma | 20           | VHL loss is the most common genetic alteration event in ccRCC, but its effect on epigenetic landscape has not been elucidated previously. We describe the genome-wide cis-regulatory landscapes of VHL-deficient ccRCC tumors by comparing the epigenetic changes in terms of histone modifications (H3K27ac, H3K4me1, H3K4me3) with the transcriptomics profiles in 10 pairs of normal kidney and ccRCC tissues. Overall design: RNA-seq profiles of 10 patient-matched normal kidney and ccRCC pairs                                                                                                                                                                                                                                                                                                                                                                                            |
| Yan et al   | PRJNA480243         | 30819200                     | Gastric Cancer                  | 21           | Total RNA was isolated from fresh frozen GC tissues using Trizol reagent (Invitrogen) according to the manufacture’s protocol. The quality of RNA was assessed using Agilent 2200 TapeStation (Agilent technologies, US) to ensure high quality (RNA integrity number > 6). Then, 1–2 μg of total RNA was used for mRNA capture using NEBNextPolyA mRNA Magnetic Isolation Module and stranded RNA-seq libraries were constructed using a NEBNext Ultra Directional RNA Library Prep Kit. The 2 × 150 paired-end sequencing was performed on an Illumina HiSeqXten (Illumina, US). The resulting raw RNA-seq files (.fastq) were preprocessed using Trimmomatic, and reads were aligned to the reference genome (GRCh37) using hisat2. Finally, the reads per kilobase per million mapped reads (RPKM) values of genes were computed to represent the expression levels of genes using StringTie. |